app/services/reblog_service.rb

```diff
@@ -17,7 +18,7 @@ def call(account, reblogged_status)
 
     return reblog unless reblog.nil?
 
-    reblog = account.statuses.create!(reblog: reblogged_status, text: '')
+    reblog = account.statuses.create!(reblog: reblogged_status, text: '', visibility: options[:visibility] || account.user&.setting_default_privacy)
```

Comment:
On the one hand, I'm still uneasy with that change in behavior when we don't have UI to tell the visibility of a boost nor UI to specify it when boosting.
On the other hand, I think it makes more sense than the current behavior…

Reply:
Posted a poll: https://mastodon.social/@Gargron/101749616287239685